pipeline/quantize/export.sh

```diff
@@ -30,13 +31,22 @@ shortlist_bin="${output_dir}/lex.50.50.${SRC}${TRG}.s2t.bin"
 "${BMT_MARIAN}"/marian-conv \
   --shortlist "${shortlist}" 50 50 0 \
   --dump "${shortlist_bin}" \
-  --vocabs "${vocab}" "${vocab}"
+  --vocabs "${vocab_src}" "${vocab_trg}"
 pigz "${shortlist_bin}"
 
-vocab_out="${output_dir}/vocab.${SRC}${TRG}.spm"
-cp "${vocab}" "${vocab_out}"
-pigz "${vocab_out}"
-
+if cmp --silent "${vocab_src}" "${vocab_trg}"; then
+  echo "Vocab files are identical, output a joint vocab"
```

Comment:
I was somewhat concerned with the artifacts duplicating the vocab in the pipeline, and it being confusing when inspecting the tasks. However, here at the final export only the shared vocab is output, so I think this will work.